From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Mon, 17 Sep 2018 21:50:15 +0200
Subject: ungoogled-chromium: Disable Network Time Tracker

This connects to Google to check if the system time is correct when a website certificate
date seems incorrect, according to https://bugs.chromium.org/p/chromium/issues/detail?id=725232

Fixes https://github.com/Eloston/ungoogled-chromium/issues/302
---
 components/network_time/network_time_tracker.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/network_time/network_time_tracker.cc b/components/network_time/network_time_tracker.cc
--- a/components/network_time/network_time_tracker.cc
+++ b/components/network_time/network_time_tracker.cc
@@ -273,7 +273,7 @@ void NetworkTimeTracker::UpdateNetworkTime(base::Time network_time,
 }
 
 bool NetworkTimeTracker::AreTimeFetchesEnabled() const {
-  return base::FeatureList::IsEnabled(kNetworkTimeServiceQuerying);
+  return false;
 }
 
 NetworkTimeTracker::FetchBehavior NetworkTimeTracker::GetFetchBehavior() const {
--
2.25.1
